tutorial/source/gp.ipynb

```diff
@@ -433,44 +436,188 @@
fe34oD2aFmOybYdO3rbyyX4WY2XeFbqy3O5DLMKQMN0gFotFGe8SstiUltzDw0N969Ytq6U4JtaAyyL9mufzuV4sFmUgYJ3DeULuKefPfT3iWWuI4s7ImcieMSOu+Rr8+4ODg42gAbkSTs1dGjLRSMsTdyq2GtvKm3J9HnxsqaDM70iLsfm5kC0oAPqAmXvV3I2JDcDRev8FXNuWW1iK89DVbt1qtdryAfYtGE3jC1uC+Xs2n+KQWAOt7WJdzqM8h+dc7Ml7SrG/z7ukEMUdYduiqRMMxp/n1ay0gB4eHuqjoyOtlNJvvvmmJiK9Xq83vlfnLHNz4LdF3c7n83IgYVF648aNSh/G0MGQO+ytW7e2JqCqvJOy/LaTuADIRa7JnQWxDKqbTqfloncymZSWpFDLqNkPcmRscZW9DYHjus/JyUkrfbwry3QXtLUb0nZdxjyXLNPV1ZVeLpdbAphPXJTfSY01kDtDNgNPzv40pMUdRHGHyIZWJ22YzdrJuULl6pStSpyNIedWi8tKrfVm9gkeBMbjsZ5Op1lWtqGi1lcuLv94PG4kGAoArd2Te2paJe7TtsXuYrHQ5+fnQUK37iI9VTyEXDsV231kwG/TfXxIsQhtvNMqYdZ1fadmjvHVme2aq9VKn5+f68lkoieTib5161ZpdGrieYe0uNMaorhzcmxX2iLdF4uFvnv3bplTUTZoW+BaHWyWYtNn+fr6uvTT5fLIrSk5OcekOeKJjq/pc39wfZdX9Ox3baa5ASAntsk9ZeLhz/AhNaenp1t/v7q6CjrZzuXONZ/Po06j5H9X+eS3ZXmS93EdCIQ+noc23qlvvuxavMWK4pDP255pPp9vuEtwIGATx7eHlnOfgCjuEJuYtK0KTf+lkAYZMnikDF4+USp/lnkdzaA2+V3fcc42XNZe07cyNFCOLWJHR0cb1nQp1EE1Qxs462Lrn7Z+WVWv8pAan0++XJzG9P2qRbsphmN88quunSuvMN/n6OgoaSwF4TTpkx4yb7W12PKVL7cot+mE8/PzjR0PNjqhLdcHorgjbB2I8/eaq0LzkItQ4ejL1sCTgS3Zva9juUSpeY/z83OtlNKPHj3yRqumDGIp4kEiPyut5icnJ0F1ALbp2kqzS/javLTqys+yz7+s16pUSa53ErpLFNo35edCffJjBE5Km5L1xs+871HzXRMzljfhasB0GSzalCi3PdO+B8V2BURxR7i2KyeTSZmihS2uMdadqsHDFnRmm3B9VHV8c/Kt8l1O6dy2YJlQMWtat8zMFSCNLq00u4Kvf8p+IwWcKX75s9JHmF2RZMCsbZcpxCWiqpw2fIGuvmtXLczrtKmh5Vftktj2Evv5mNSfXfsd5xartmdq4zmvrraPqi6K8GOpdxWI4p5RFJ+laJG5RkM7WuoKPNTqyn/j8pycnGxlrlitVnq5XG5t2/qCbmItxb4UbCF1cn19rafTqTV/JCbNdGC98OPqW6a7jynoTOvu2dnZhtWTRfLx8XH5s2+XyPUZLmPMZCj743Q6tR49K78XuzBPbVNDO4mrS1JEWO5FdEj7Ttl9SAkizfVM5k5sUYTtHscYx1zPxP3aDJLf93SlEMU9gq0m5il0PGH6fI/rDPbmpFNlzZIJx9npX0apm6dhuQaelAHKVpYqH0bXfY6OjpKtzWCb3BPCkLBNUuZOS1VgXlVmhVCx6+ovtnGHLc/8N5m1gq9dld7N127qtCn53ZQUczLwV9ZFSEAwCCPnIrpK6JlBoXx4RtUhNaHzVMp8VlX24+Pjrbl0Op3q5XLpfE6zPFVulFXPZDuAZJ+BKO4IWydYr9elGJb+eeZRkKGiMwRXx/H9Xk6+8mQdU7RXTSYp2zNmvUmLdcxz8uLDFpG/S7lY+0KdCQFsw/VnulJIgcruACcnJ6WbFbsVuRLshwpe7ucyPaHtuzaxaR5CVOXrrLU/8DClTZmfTckDz5/h8pv/76pt78vY0vYiWr5zbm9ybq1b1jrvxdfW69RTjoB7vsZQdgAhijvC1gmm0+mWT/FoNNLn5+cb341xT4gtg/zZ1qFM9wnOS2zz8Q05DSjHxBcbSW/6GDYx2XHZjo+PS6sZC4jr62t9dnZWfm7XJjMb+zJR9wHZB1w+t7x7xNbkyWRS5iOvCiqz9RtbP7SdtBU7kZq7TzHW4FzZJ5hQf2pbXZnHw3e52AsdM/vcJ3MsolNdNmItn6457+zsLHv7dJ1om+I+KduuS/yHZJeZTqd6PB5vuRr2oR01AURxh5iTAVtrzM7n2hYJDWRz4RtUYiYxVyBgUWy6Wrj8kVJWwqaFy1xZ+7aSpHVL1qWZ57HupFIUmzkleVJmd5OUiaBp+jyRtkEfnj+0DGz95DzFtgVfqGjle8htZe63sRO0q6/JRanpOpEijlLeVcp2PX+H69n13TbbTsiYmUN4xhDjbpKjrlKez1zsSWOFqywspOXuqJm+M/T+vrLPZjN9enq6tZg03Serni/U4FPVhvi55eFftqOq9w2I4o6JsahI5ISYO9WQz8JrduaqbUk5CMnAwap6qML0D+N/2/ITVw2ernvnmFTk1hNPqrHWqhByTchtT6R9w7XYYncF0/dWBp8ybN2VliSXOKhbTl4Yv/nmm3oymWxM9r7AONcYY9tWNvtY1URqGx/W67XVHaSpLWff55uyFLfdd0LGzJRnTiVGkJmktgMWk6FucDy3yfpgY4XrvRWF27hTp37N75onzbIIjT1wpiheGWPM8wFknbBV2ndd6d4o+8F0Oq30w95lIIo7xOwUoStPc0J0DdSpA40t4pUDElwCwCYW+N8232hfPcQ+R53vh07yVYNe1XbtwcFBKYybyNAQMyHXrU8bfbCw5oIXgTK7CZ8aJY9R5j7ryhzBQrAJcXB1dVUm8OdxYDqd6qOjIz2dTp3bwr52wv9mH2VzYWk+p2siNe/LsRJNpEYLbaspglW+49FopN97772N8q/X69INKrY8dYm5T5sZYWIWEbbv
```

Comment:
Why is the envelope so narrow around the inducing points? I would expect a plot of the estimated total variance at each point x, whereas it looks like you're plotting only the uncertainty in the mean (not the variance due to measurement error). Do you think it would be clearer to plot total uncertainty rather than uncertainty in the mean?

Reply:
(this comment refers to the final figure)

Reply:
Nice catch! The current implementation predicts $f*$, not $y*$. Although I have put the option: `noiseless=False`, I forgot to include it in the implementation. Will fix this soon!